src/OpenSage.Game/Assets/Shaders/Sprite.frag

```diff
@@ -74,11 +74,11 @@ void main()
 
     // Currently we always sample from the alpha mask,
     // falling back to a default one if there isn't a real one.
-    ivec2 alphaMaskSize = textureSize(AlphaMask, 0);
+    ivec2 alphaMaskSize = textureSize(sampler2D(AlphaMask, Sampler), 0);
```

Comment:
I seems like the `GL_EXT_samplerless_texture_functions` - extension does not work for OpenGL then?

Reply:
I am not sure, I just checked my RX 580 and gtx1070 and neither of them has that extension, so perhaps that explains why glslangValidator created a dummy texture object AlphaMask_1 that was never bound.

Reply:
ok, i think we should remove that extension at the top then, since we do not use it anymore in that file.